docs/deployment/MIGRATIONS.md

```diff
@@ -0,0 +1,32 @@
+# Running commands on staging instances
+
+The following instructions can be used to apply migration commands on an AWS stack of this project.
+
+## Prerequisites
+
+- Access to the AWS Web Console
+
+## Console commands
+
+ - First, log into the [AWS Web Console](https://raster-foundry.signin.aws.amazon.com) and navigate to the [EC2 Container Service](https://console.aws.amazon.com/ecs/home?region=us-east-1#/clusters) dashboard.
```

Comment:
The first link here should be https://raster-foundry.signin.aws.amazon.com/console.